static/js/echo.js

```diff
@@ -478,7 +482,8 @@ $(function () {
         userMentionHandler: handleUserMentions,
         realmFilterHandler: handleRealmFilter,
         renderer: r,
-        preprocessors: [preprocess_code_blocks]
+        preprocessors: [preprocess_code_blocks],
+        streamLinkHanlder: handleStreamLinks
```

Comment:
maybe put this next to the other similar custom handlers in the list?  Also, `Handler` is mispelled.